src/amber/cli/commands/database.cr

```diff
@@ -19,6 +19,8 @@ module Amber::CLI
       end
 
       def run
+        migrations_path, migrations_table_suffix = Micrate::Cli.parse_command_arguments
```

Comment:
Why was this necessary. Looks like we're just passing commands back into Micrate that it gave us in the same scope.

Reply:
I just fixed it to work with latest version of micrate shard see: https://github.com/juanedi/micrate/commit/af301fcec0c42ee5872fe0d783fbf91a762a11b9

Reply:
I reverted this changes, please check this PR again 😄